static/app/utils/performanceForSentry.tsx

```diff
@@ -43,18 +43,27 @@ export const VisuallyCompleteWithData = ({
   useEffect(() => {
     try {
       const transaction: any = getCurrentSentryReactTransaction(); // Using any to override types for private api.
-      const now = performance.now();
+      const now = timestampWithMs();
+      const transactionStart = transaction.startTimestamp;
+      const normalizedValue = Math.abs((now - transactionStart) * 1000);
+
       if (!isVisuallyCompleteSet.current) {
-        transaction.setMeasurements({
-          ...transaction._measurements,
-          visuallyComplete: {value: now},
+        transaction.registerBeforeFinishCallback(function (t, _) {
```

Comment:
Any reason why don't use arrow functions here? There shouldn't be any lexical scope concerns right?

Reply:
Yeah arrow would work fine, got this from a debugger breakpoint, likely reading transpiled version, so yeah.